server/schemas/field.graphql

```diff
@@ -0,0 +1,104 @@
+enum FiledType {
+  Text
+  TextArea
+  RichText
+  MarkdownText
+  Asset
+  Date
+  Bool
+  Select
+  Tag
+  Integer
+  Reference
+  URL
+}
+
+type SchemaField {
+  id: ID!
+  modelId: ID!
+  model: Model!
+  typeProperties: SchemaFieldTypeProperties
+  key: String!
+  title: String!
+  description: String
+
+  # FieldSettings
+  multiValue: Boolean
+  defaultValue: Any
+
+  # FieldConstraints
+  unique: Boolean
+  required: Boolean
+  #  max: Int
+  #  min: Int
+
+  createdAt: DateTime!
+  updatedAt: DateTime!
+}
+
+union SchemaFieldTypeProperties = SchemaTextFieldProperties | SchemaTextAreaFieldProperties | SchemaRichTextFieldProperties | SchemaMarkdownTextFieldProperties | SchemaAssetFieldProperties | SchemaDateFieldProperties | SchemaBoolFieldProperties | SchemaSelectFieldProperties | SchemaTagFieldProperties | SchemaIntegerFieldProperties | SchemaReferenceFieldProperties | SchemaURLFieldProperties
+
+
+type SchemaTextFieldProperties  { _f: Any}
+type SchemaTextAreaFieldProperties  { _f: Any}
+type SchemaRichTextFieldProperties  { _f: Any}
+type SchemaMarkdownTextFieldProperties  { _f: Any}
+type SchemaAssetFieldProperties { _f: Any}
+type SchemaDateFieldProperties  { _f: Any}
+type SchemaBoolFieldProperties  { _f: Any}
+type SchemaSelectFieldProperties {
+  values: Any
+}
+type SchemaTagFieldProperties  { _f: Any}
+type SchemaIntegerFieldProperties  { _f: Any}
+type SchemaReferenceFieldProperties {
+  referencedModelId: ID
+}
+type SchemaURLFieldProperties  { _f: Any}
```

Comment:
```suggestion
union SchemaFieldTypeProperty = 
    SchemaFieldText
  | SchemaFieldTextArea
  | SchemaFieldRichText
  | SchemaMarkdownText
  | SchemaFieldAsset
  | SchemaFieldDate
  | SchemaFieldBool
  | SchemaFieldSelect
  | SchemaFieldTag
  | SchemaFieldInteger
  | SchemaFieldReference
  | SchemaFieldURL

type SchemaFieldText {
  defaultValue: String
  maxLength: Int
}

type SchemaFieldTextArea {
  defaultValue: String
  maxLength: Int
}

type SchemaFieldRichText {
  defaultValue: String
  maxLength: Int
}

type SchemaFieldMarkdownText {
  defaultValue: String
  max: Int
}

type SchemaFieldAsset {
  assetType: AssetType # for example
}

type SchemaFieldDate {
  _: Any
}

type SchemaFieldBool {
  defaultValue: Boolean
}

type SchemaFieldSelect {
  values: [String!]!
  defaultValue: String
}

type SchemaFieldTag {
  values: [String!]!
  defaultValue: String
}

type SchemaFieldInteger {
  defaultValue: Int
  min: Int
  max: Int
}

type SchemaFieldReference {
  referencedModelId: ID
}

type SchemaFieldURL {
  defaultValue: String
}
```

- Shorten names
- Format graphql file
- DefaultValue can be put at each SchemaFieldTypeProperty